templates/chatroom/grafbase/schema.graphql

```diff
@@ -0,0 +1,29 @@
+type Room @model {
+  title: String!
+  private: Boolean @default(value: false)
```

Comment:
don't booleans have a default value already?

Reply:
If it's nullable then it's `null`. Otherwise if it's non-nullable `Boolean!` then a value is required on create.